nncf/quantization/algorithms/min_max/torch_fx_backend.py

```diff
@@ -177,16 +177,25 @@ def _get_input_scale_shape(
         nncf_graph: NNCFGraph, target_point: PTTargetPoint, per_channel: bool
     ) -> Tuple[Tuple[int, ...], Tuple[int, ...], int]:
         is_weights = target_point.is_weight_target_point()
+        input_shape = nncf_graph.get_input_shape_for_insertion_point(target_point)
+
         if is_weights:
-            # TODO(dlyakhov): support transpose conv/ make channel_idx common
-            channel_idx = 0
+            node = nncf_graph.get_node_by_name(target_point.target_node_name)
+            channel_axes = get_weight_channel_axes(node.metatype, len(input_shape), target_point.input_port_id)
         else:
-            channel_idx = 1  # channel dim for activations
+            channel_axes = [1]
 
-        input_shape = nncf_graph.get_input_shape_for_insertion_point(target_point)
-        scale_shape = tuple(
-            get_scale_shape(input_shape, is_weights=is_weights, per_channel=per_channel, channel_idx=channel_idx)
-        )
+        channel_idx = channel_axes[0] if channel_axes else 0
+
+        if is_weights and not channel_axes:
```

Comment:
```suggestion
        if not len(channel_axes):
```
to cover the case of vector weights which are being quantized per channel